pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -149,6 +145,28 @@ func (p *LaunchTemplateProvider) ensureLaunchTemplate(ctx context.Context, optio
 	return launchTemplate, nil
 }
 
+func needsGPUAmi(is []cloudprovider.InstanceType) bool {
```

Comment:
i might've misled you earlier. Should these live in ami.go (where they're used)?